src/Run/Setup.cs

```diff
@@ -150,10 +150,7 @@ private string BuildParametersForCommand(Dictionary<string, string> commandParam
         {
             string commandSetting = string.Empty;
 
-            if (Tools.ContainsKey(toolName))
-            {
-                commandSetting = Tools[toolName].osSpecific[Os]["defaultParameters"];
-            }
+            Tools[toolName].osSpecific[Os].TryGetValue("defaultParameters", out commandSetting);            
```

Comment:
Yes, the BuildParametersForCommand function is called by the function [BuildCommand](https://github.com/dotnet/buildtools/pull/972/files/3d163b9d9d81bd9d3b7439ac4a0cf8104a57c2cb..2b2df73d2267fbb83824eb5700f0c1338f4ae497#diff-6f4e4ec63e7b18f5f61e1d0ba5c10087R114) which verifies that the toolName is in Tools.